hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/ha/SCMHAConfiguration.java

```diff
@@ -38,7 +38,7 @@
 
   @Config(key = "ratis.storage.dir",
       type = ConfigType.STRING,
-      defaultValue = "",
+      defaultValue = "scm-ratis",
```

Comment:
This is workaround to config SCM ratis storage dir value. The usual way that config value in SCMHAConfiguration seems break. I will file a JIRA to fix it.

Reply:
https://issues.apache.org/jira/browse/HDDS-4624

Reply:
We won't need this line of change after https://github.com/apache/ozone/pull/1739